Orange/widgets/data/owpreprocess.py

```diff
@@ -252,35 +252,103 @@ def __repr__(self):
 
 class RemoveSparseEditor(BaseEditor):
 
+    options = ["Nan's", "0's"]
```

Comment:
NaNs and 0s, no apostrophe. I prefer missing instead of NaN, but that's ok, too. 0 should probably be written with a word, 'zeros'.

Reply:
I think we should use "missing". Muggles won't understand NaN.